server/chat-plugins/modlog.ts

```diff
@@ -46,6 +46,12 @@ const MAX_RESULTS_LENGTH = MORE_BUTTON_INCREMENTS[MORE_BUTTON_INCREMENTS.length
 const LOG_PATH = 'logs/modlog/';
 const IPS_REGEX = /[([]([0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3})[)\]]/g;
 
+const GLOBAL_PUNISHMENTS = [
+	'WEEKLOCK', 'LOCK', 'BAN', 'RANGEBAN', 'RANGELOCK', 'FORCERENAME',
+	'TICKETBAN', 'AUTOLOCK', 'AUTONAMELOCK', 'NAMELOCK', 'AUTOBAN', 'MONTHLOCK',
+];
+const GLOBAL_PUNISHMENTS_REGEX_STRING = `\\b(${GLOBAL_PUNISHMENTS.join('|')}):.*`;
+
 const PUNISHMENTS = [
```

Comment:
this should probably `..GLOBAL_PUNISHMENTS,`

Reply:
yea it probably should